packages/components/src/Disclosure/docs/description.tsx

```diff
@@ -0,0 +1,20 @@
+import { Disclosure, DisclosureHeader, DisclosurePanel, Div, Flex, Text } from "@hopper-ui/components";
+
+export default function Example() {
+    return (
+        <Div width="100%" paddingX="core_320" paddingY="core_480">
+            <Disclosure>
+                <DisclosureHeader>
+                    <Flex wrap="wrap" columnGap="inline-sm" alignItems="baseline">
```

Comment:
There is the Inline component that should be used here i think

Reply:
I guess that would work too